e2e/test/scenarios/native/reproductions/30680-native-editor-buttons.cy.spec.js

```diff
@@ -13,7 +13,7 @@ describe("issue 30680", () => {
       .findByText("Use a native query")
       .click();
 
-    cy.get(".ace_editor").type("select * from orders ");
+    cy.get(".ace_editor").type("select * from orders ", { delay: 100 });
```

Comment:
unrelated to my changes but this was failing consistently on my branch for some reason 🤷‍♂️